### PR TITLE
Enable issue templates

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
This commit adds templates for bug report, feature request and support
related issues.

did this on the operator side, thought is also a good idea for the Operand

Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+name: Bug Report
+about: Report a bug encountered while operating the Node-Feature-Discovery
```
Comment (marquiz) [inline .github/ISSUE_TEMPLATE/bug-report.md:3]: nit:
```suggestion
about: Report a bug encountered while using Node Feature Discovery
```

File: .github/ISSUE_TEMPLATE/enhancement.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: Enhancement Request
+about: Suggest an enhancement to the Node-Feature-Discovery project
```
Comment (marquiz) [inline .github/ISSUE_TEMPLATE/enhancement.md:3]:
```suggestion
about: Suggest an enhancement to the Node Feature Discovery project
```

File: .github/ISSUE_TEMPLATE/support.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+name: Support Request
+about: Support request or question relating to Node-Feature-Discovery
```
Comment (marquiz) [inline .github/ISSUE_TEMPLATE/support.md:3]:
```suggestion
about: Support request or question relating to Node Feature Discovery
```

**Comment by marquiz** [comment]
Thanks @ArangoGutierrez !
/lgtm

**Comment by ArangoGutierrez** [comment]
> continuous-integration/travis-ci Expected — Waiting for status to be reported 
Ah?

**Pull request merged**